src/os/ui/slick/abstractgroupbytreesearch.js

```diff
@@ -78,8 +78,6 @@ os.ui.slick.AbstractGroupByTreeSearch.prototype.searchNodes = function(exp, resu
       }
     }
   }
-
-  this.finalizeSearch(groupBy, results);
```

Comment:
This call happens twice currently in treesearch.js, one right after the other. In `beginSearch`, it calls `this.searchNodes` (which calls `this.finalizeSearch`), and then directly calls `this.finalizeSearch` again. We have several places that use these core search classes and this poses a risk of breaking them, but the double call is definitely bad and I wanted to clean it up.